### PR TITLE
Use Symfony Filesystem component to do remove and rename operations.

### LINKED ISSUE #108: Building the website does not work on windows
While a nice-to-have, being unable to build the website on windows might make contributions harder for some people (and yes, I tried contributed from my Windows computer rather than the Ubuntu one I'm using for development most of the time).

Until now I identified these issues:

- sculpin has a bug on windows, breaking the CLI initialization (fixed in https://github.com/dflydev/dflydev-embedded-composer/pull/22)
- the `build-website` command fails when trying to run processes using globs with `mv`

### PR DESCRIPTION
**Pull request by jwage**
Fixes #108

### CODE REVIEW

**Comment by jwage** [comment]
@stof would you mind testing this on Windows pretty please? :)

**Comment by jwage** [comment]
Bah its already out of date. I will update later today.

**Comment by stof** [comment]
I'm also getting weird Twig errors when performing the website building on Windows. I haven't tried it on Linux to compare (my Linux computer is not ON currently).

**Comment by jwage** [comment]
@stof does windows not support php.net/glob?

File: app/src/Doctrine/Website/WebsiteBuilder.php [hunk 1]
```diff
@@ -86,10 +93,10 @@ public function build(
         $outputDir = sprintf('output_%s', $env);
 
         // cleanup the build directory
-        $this->processFactory->run(sprintf('rm -rf %s/*', $buildDir));
+        $this->filesystem->remove(glob($buildDir . '/*'));
 
         // copy the build to the build directory
-        $this->processFactory->run(sprintf('mv %s/%s/* %s', $rootDir, $outputDir, $buildDir));
+        $this->filesystem->mirror($rootDir . '/' . $outputDir, $buildDir);
```
[the remark this answers is not in the document]
Comment (jwage) [reply app/src/Doctrine/Website/WebsiteBuilder.php]: Hmm, `$rootDir . '/' . $outputDir` should exist from the `./vendor/bin/sculpin generate` that gets called before we try to call `mirror()`. Is something failing before that?

**Comment by stof** [comment]
@jwage this is supported. but RstBuilder uses `shell_exec` with a command containing a glob. This is not a PHP glob. It is a shell one (and this one does not work the same)

**Comment by jwage** [comment]
Oh I see. I will update that.

**Comment by jwage** [comment]
@stof I think I got rid of all of the raw glob and shell_exec calls that won't work on windows.

**Comment by jwage** [comment]
I updated to use Symfony Filesystem in https://github.com/doctrine/doctrine-website/pull/207 but I need someone with Windows to verify it works.

**Pull request closed without merging**